tests/test_helper/setup_teardown/tls_cluster.bash

```diff
@@ -0,0 +1,31 @@
+setup_file() {
+    load test_helper/common.bash
+    load test_helper/lxd.bash
+    load test_helper/microovn.bash
+
+
+    TEST_CONTAINERS=$(container_names "$BATS_TEST_FILENAME" 4)
+    CENTRAL_CONTAINERS=""
+    CHASSIS_CONTAINERS=""
+    counter=0
+    # Categorize first three containers as "CENTRAL" and reset as "CHASSIS"
+    for container in $TEST_CONTAINERS; do
+        if [ $counter -lt 3 ]; then
```

Comment:
Assuming the first three containers will always have the central services may not hold true.

Why not just check what services are in each container when running the tests?

Reply:
+1. The helper function for checking container services was not available during the initial implementation.

Reply:
Replaced with suggestion